releases/release-1.31/release-notes/maps/pr-126139-map.yaml

```
@@ -0,0 +1,5 @@
+pr: 126139
+releasenote:
+  text: Reverted "Move ConsistentListFromCache feature flag to beta and enable it
+    by default".
```

```suggestion
  text: Reverted [feature](https://github.com/kubernetes/kubernetes/pull/123513) where `ConsistentListFromCache` was moved to beta and enabled by default".
```

---

```suggestion
  text: Reverted a [change](https://github.com/kubernetes/kubernetes/pull/123513) where `ConsistentListFromCache` was moved to beta and enabled by default.
```

I wonder if we can make these two entries cancel out, somehow?